### PR TITLE
Adds Firefighter Helmets to Autolathe

### PR DESCRIPTION
**Pull request by TerraGS**
## About The Pull Request
Adds Firefighter Helmests (red hardhats) to the list of items Autolathes can print.

## Why It's Good For The Game
With emergency fire lockers regularly looted and the recent addition of firebots which require this type of hardhat, making it printable reduces competition over them and having to hunt down a non-looted locker.

## Changelog
:cl:
add: Adds Firefighter Helmets to Autolathes
/:cl:

### CODE REVIEW

**Comment by anconfuzedrock** [comment]
This is making the autolathe able to print hard hats
"ok guys I'm gonna add a new barbot that needs a shaker and a shotgun"
"With bar closets regularly looted and the recent addition of barbots which require this type of gun, making it printable reduces competition over them"
If the thing needing a firehat is a problem then making it not need a firehat is the solution

**Comment by FantasticFwoosh** [comment]
I half agree, we have different variants in the code of the same object, you could have easily have just printed a blue workhat, or a yellow standard workhat with a light affixed rather than the special one that **so happens to also be pressure & fire proof**. But firefighting gear in itself is meant to be rare in the same way space suits are, as to create competition and kill off people unable to seize supplies in time in a emergency. 

- Nomex (preferably made via chemistry similar to plastic, real material related to kevlar) that is fireproof and makes fireproof objects directly, or through further manufaction. Even simple to say 10 nomex sheets makes 1 fire or radiation suit.

- Firefighting gear could be added with RnD nodes, the Nomex additional cost justifying making it and later more generally expensive atmos-tier firefighting kit. (firefighting helmet inserted here)

Either of these solutions could keep firefighting bot as is without this PR.

**Comment by TerraGS** [comment]
If this doesn't go through I'll just change firebots when I fix the runtimes.

**Comment by nemvar** [comment]
My sprites suck anyways. Wouldn't mind if someone replaced them. Just thought it would be kinda funny if it looked like a fire hydrant.

**Comment by nemvar** [comment]
Also there are far more fire helmets on meta than there are medkits. You can get 5 helmets without even leaving science/science maint. That are WAY more helmets than the average amount of firebots constructed each round.

TL;DR I can't sprite for shit and this is a non issue otherwise.
